R/joyn-merge.R

```diff
@@ -539,7 +539,43 @@ joyn <- function(x,
 
   # return messages
   joyn_report(verbose = verbose)
-  if (verbose == TRUE) joyn_msg(msg_type)
+
+  if (verbose == TRUE) {
+    #joyn_msg(msg_type)
+
+    notes_count <- sum(
+      .joynenv$joyn_msgs$type %in% c("info", "note")
+      )
+
+    warn_count  <- sum(
+      .joynenv$joyn_msgs$type == "warn"
+      )
+
```

Comment:
Hi @RossanaTat , 

thank you so much. 

What happens when `.joynenv$joyn_msgs$type`is NULL or has NA? I suggest you retrieve `joyn_msgs` with `rlang::env_get()` and use either `sum( na.rm = TRUE)` or `fsum()` from collapse. 

Thanks.

Reply:
Hi @randrescastaneda, thank you for reviewing. I have pushed some changes following your suggestions, and also to show the user the clickable messages only if there are any. Let me know if it looks okay now. Thank you!